packages/testsuite-e2e_native/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "ts-tests",
+  "name": "testsuite-e2e_native",
```

```suggestion
  "name": "@defimetachain/testsuite-web3-rpc",
```

Let's call it web3-rpc, since we're testing the web3 RPCs.

---

- [ ] remember to rename the directories
- [ ] fix the conflicts